src/symbols.c

```diff
@@ -1536,16 +1456,7 @@ static void show_goto_popup(GeanyDocument *doc, GPtrArray *tags, gboolean have_b
 	if (first) /* always select the first item for better keyboard navigation */
 		g_signal_connect(menu, "realize", G_CALLBACK(gtk_menu_shell_select_item), first);
 
-	event = gtk_get_current_event();
-	if (event && event->type == GDK_BUTTON_PRESS)
-		button_event = (GdkEventButton *) event;
-	else
-		gdk_event_free(event);
```

Comment:
BTW: This looks like a memory leak. The event object is only freed if it is triggered by keyboard. Good thing that we'll get rid of it soon :slightly_smiling_face: